diff -c -r /home/swetha/repos/captain_oveur/test/test_app/app/controllers/application_controller.rb ./app/controllers/application_controller.rb
*** /home/swetha/repos/captain_oveur/test/test_app/app/controllers/application_controller.rb	2010-07-23 12:33:16.993512218 -0400
--- ./app/controllers/application_controller.rb	2010-07-23 12:00:46.566202061 -0400
***************
*** 6,12 ****
    include CaptainOveur::Authentication
    helper :all # include all helpers, all the time
    protect_from_forgery # See ActionController::RequestForgeryProtection for details
! 
    # Scrub sensitive parameters from your log
    # filter_parameter_logging :password
  end
--- 6,12 ----
    include CaptainOveur::Authentication
    helper :all # include all helpers, all the time
    protect_from_forgery # See ActionController::RequestForgeryProtection for details
!   layout 'home'
    # Scrub sensitive parameters from your log
    # filter_parameter_logging :password
  end
diff -c -r /home/swetha/repos/captain_oveur/test/test_app/config/environments/cucumber.rb ./config/environments/cucumber.rb
*** /home/swetha/repos/captain_oveur/test/test_app/config/environments/cucumber.rb	2010-07-23 12:32:53.909511667 -0400
--- ./config/environments/cucumber.rb	2010-07-23 09:28:38.517514587 -0400
***************
*** 25,27 ****
--- 25,49 ----
  config.gem 'database_cleaner', :lib => false, :version => '>=0.5.0' unless File.directory?(File.join(Rails.root, 'vendor/plugins/database_cleaner'))
  config.gem 'webrat',           :lib => false, :version => '>=0.7.0' unless File.directory?(File.join(Rails.root, 'vendor/plugins/webrat'))
  
+ config.gem 'factory_girl'
+ 
+ config.action_mailer.default_url_options = { :host => 'example.com' }
+ 
+ class RackRailsCookieHeaderHack
+  def initialize(app)
+    @app = app
+  end
+ 
+  def call(env)
+    status, headers, body = @app.call(env)
+    if headers['Set-Cookie'] && headers['Set-Cookie'].respond_to?(:collect!)
+      headers['Set-Cookie'].collect! { |h| h.strip }
+    end
+    [status, headers, body]
+  end
+ end
+ 
+ config.after_initialize do
+  ActionController::Dispatcher.middleware.insert_before(ActionController::Base.session_store, RackRailsCookieHeaderHack)
+ end
+ 
diff -c -r /home/swetha/repos/captain_oveur/test/test_app/config/environments/test.rb ./config/environments/test.rb
*** /home/swetha/repos/captain_oveur/test/test_app/config/environments/test.rb	2010-07-23 12:32:32.129513208 -0400
--- ./config/environments/test.rb	2010-07-23 09:19:21.357512056 -0400
***************
*** 25,28 ****
  # Use SQL instead of Active Record's schema dumper when creating the test database.
  # This is necessary if your schema can't be completely dumped by the schema dumper,
  # like if you have constraints or database-specific column types
! # config.active_record.schema_format = :sql
\ No newline at end of file
--- 25,30 ----
  # Use SQL instead of Active Record's schema dumper when creating the test database.
  # This is necessary if your schema can't be completely dumped by the schema dumper,
  # like if you have constraints or database-specific column types
! # config.active_record.schema_format = :sql
! 
! config.action_mailer.default_url_options = { :host => 'localhost:3000' }
